src/middlewared/middlewared/plugins/nfs_/status.py

```diff
@@ -0,0 +1,32 @@
+from psutil._pslinux import Connections
+
+from middlewared.service import Service
+
+
+class NFSService(Service):
+    def client_count(self):
+        """
+        Return currently connected clients count.
+        """
+
+        """
+        As NFS does not have an explicit umount request, Ganesha does not remove clients and the following snippet
+        does not work correctly:
+
+        bus = dbus.SystemBus()
+        proxy = bus.get_object("org.ganesha.nfsd", "/org/ganesha/nfsd/ClientMgr")
+        iface = dbus.Interface(proxy, dbus_interface="org.ganesha.nfsd.clientmgr")
+        return len(iface.ShowClients()[1])
+
+        We should reconsider our approach if they ever fix this.
+        """
+
+        with open("/var/run/ganesha.pid") as f:
```

Comment:
I know that we check if the service is running 1st, but probably best to catch FileNotFoundError and return early.

Reply:
We catch all these exceptions in event handler (and ignore them in production mode) so I thought it would be fine for this method to fail.